docs/src/app/components/code-example/code-block.jsx

```diff
@@ -2,12 +2,14 @@ const React = require('react');
 const ReactDOM = require('react-dom');
 const { Styles } = require('material-ui');
 const { Spacing } = Styles;
+const { ThemeManager } = Styles;
+const DefaultRawTheme = Styles.LightRawTheme;
```

Comment:
eslint was throwing me an error later down the file because Thememanager and DefaultRawTheme weren't defined. looked at what the other files in docs were doing to handle it and slapped it in there.